kratos/sources/parallel_environment.cpp

```diff
@@ -130,6 +144,9 @@ void ParallelEnvironment::PrintData(std::ostream &rOStream)
 ParallelEnvironment::ParallelEnvironment()
 {
     RegisterDataCommunicatorDetail("Serial", DataCommunicator::Create(), MakeDefault);
+    RegisterCommunicatorFactoryDetail([](ModelPart& rModelPart, const std::string& rDataCommunicatorName)->Communicator::UniquePointer{
+        return Communicator::UniquePointer(new Communicator());
+    });
```

Comment:
so when running in serial it always gives back a `Communicator` and ignores the name?

Reply:
Genau. This is something I'd like to discuss in the next @KratosMultiphysics/technical-committee meeting.